tests/v0_tests/test_bulk_search.py

```diff
@@ -68,11 +69,15 @@ def strip_marqo_fields(doc, strip_id=True):
     @mock_get_indexes_response([GetIndexesIndexResponseObject.get_default_index_object()])
     def test_bulk_search_with_context(self):
         """Check that context is passed to HTTP request correctly"""
+        marqo_url_and_version_cache.clear()
+        cache_url = self.client.config.instance_mapping.get_index_base_url(self.generic_test_index_name)
+        marqo_url_and_version_cache[cache_url] = "v0"
```

Comment:
add a comment about why you are doing this, within the context of the test (we are simulating that the version is already checked, for example.